week1/practice/ComposeQuadrant/app/src/main/java/com/example/composequadrant/MainActivity.kt

```diff
@@ -41,10 +72,28 @@ private fun ComposableInfoCard(
     backgroundColor: Color,
     modifier: Modifier = Modifier
 ) {
-    Column( ) { }
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(backgroundColor)
+            .padding(16.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Center
```

Comment:
```suggestion
        modifier = modifier
            .fillMaxSize()
            .background(backgroundColor)
            .padding(16.dp),
        horizontalAlignment = Alignment.CenterHorizontally,
        verticalArrangement = Arrangement.Center
```
ComposableInfoCard의 인자로 받은 modifier을 전달해야 `weight(1f)`가 적용될 것 같습니다!